src/pyskindose/geom_calc.py

```diff
@@ -45,7 +45,17 @@ def calculate_field_size(field_size_mode, data_parsed, data_norm):
         FS_lat = round(100 * np.sqrt(data_parsed.CollimatedFieldArea_m2), 3)
         FS_long = FS_lat
 
-    return FS_lat, FS_long
+        return FS_lat, FS_long
+
+    if field_size_mode == 'ASD':
+
+        FS_long = (data_parsed.LeftShutter_mm + data_parsed.RightShutter_mm) / 10
```

Comment:
Maybe we should add some comments (or preferably an aptly named helper function) to make it clear why this is done. A helper function could, e.g., be named `convert_length_measurement_unit_to_cm(measurement_value: float, unit: str)`.

Reply:
Agree.

The function has been rebuilt.